packages/message/src/Granules.ts

```diff
@@ -252,6 +252,10 @@ export const generateGranuleApiRecord = async ({
     createdAt,
   } = granule;
 
+  const now = Date.now();
+  const recordUpdatedAt = updatedAt ?? now;
+  const recordTimestamp = timestamp ?? now;
```

Comment:
I'm still wondering if we should set this to `now` for the framework in writeGranulesFromMessage.    It would tidly solve the issue with execution ordering....   

Hmm.

Reply:
This PR does do that also still.

Reply:
Or are you saying that's bad?

Reply:
Hmm, because it's not passed at all.    Sorry, disregard.